internal/cmd/auth.go

```diff
@@ -52,5 +53,14 @@ func newClient() (*coder.Client, error) {
 		return nil, err
 	}
 
+	if !version.VersionsMatch(apiVersion) {
+		clog.Log(clog.Warn(
+			"version mismatch detected",
+			fmt.Sprintf("coder-cli version: %s", version.Version),
+			fmt.Sprintf("Coder API version: %s", apiVersion), clog.BlankLine,
+			clog.Tipf("download the appropriate version here: https://github.com/cdr/coder-cli/releases"),
```

Comment:
Just to make sure, this is thrown to stderr correct?

Reply:
Yup, all log levels are thrown in stderr.